### PR TITLE
Prism validation article

### LINKED ISSUE #67: articles/prism/validation.md

### PR DESCRIPTION
**Pull request by pytlesk4**
closes #67

### CODE REVIEW

File: articles/prism/validation.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+_Note: We plan to introduce templates to the Stoplight editor file creation process soon. This feature will automate most of the steps below and turn mock server creation into a one-click solution._
+
+_For this article we will validate a service that already exists. It is just a simple API representing a todos list manager that is running at http://todos.stoplight.io. We have created an OAS Specification for it already and you can download it [here](https://exporter.stoplight.io/3351/master/todos.oas2.yml)._
+
+1. Let's create a new project, create a new spec, name it `todos.oas2`, and paste the JSON from the spec above in the code editor.
+
+![](../../assets/gifs/validation-todos-contract-guide.gif)
+
+2. Create a new **Prism instance file** in the project. Name it `todos.contract.prism`.
+3. Prism instances are made up of APIs and Rules, you can learn more about them here. Add an API to the Prism instance and connect the `todos.oas2` specification that you created earlier. Also, let's change the `id` to `todos` and set the _Upstream URL_ to _http://todos.stoplight.io_. The Upstream URL is where the contract server will forward incoming requests.
```
Comment (rowa97) [inline articles/prism/validation.md]: link missing "you can learn more about them here"

File: articles/prism/validation.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+5. Once you have created a new rule, you need to connect it to the API we added earlier. To do that, click on the `apis` dropdown input and select the previously created API. Connecting the rule to the API you defined earlier makes the OAS file available to scenarios in the rule.
+6. Lastly, you need to add a **scenario** that will actually perform the validation. We have an official Stoplight validate scenario [here](https://next.stoplight.io/stoplight/prism?edit=%23%2Fscenarios%2validate), which makes it easy to get started. 
+    1. Add a scenario to the `after` section of your rule. 
+    2. Select `another project` in the first dropdown.  
+    3. Search for `prism`. 
+        1. The file you are looking for within that project is `helpers.prism.yml` and the specific scenario is called `validate`. 
+        
+This validate scenario should suit most of your needs. It will check the request/response headers, request/response body, request path parameters, and query strings. It will also add response headers to the HTTP request on the way back to the consumer with the results of the validation. For advanced use cases, please send us a [message](). We would love to help out!
```
Comment (rowa97) [inline articles/prism/validation.md]: add link "please send us a message"

File: articles/prism/validation.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+# Validating Mock Servers
+
+Validating an existing service is powerful, but what happens if you are still implementing your API and all you have is a mock server? How do you keep the examples valid?
+
+_Note: If you don't have an existing mock server, check out [this](https://next.stoplight.io/stoplight/stoplight-next-docs/blob/master/prism.mock.server.md) article first and then continue reading._
+
+## Steps
+
+1. Repeat step 7 above. That is it. Now you will know when your examples are out of date. Not only will your mock server be accurate, but it will help you catch any errors in examples in your documentation that you provide to users.
```
Comment (rowa97) [inline articles/prism/validation.md]: I changed the numbering convention earlier. Can you double check to make sure this is referencing the right step?

**Comment by marbemac** [comment]
Putting a hold on this until stoplightio/core#707 is completed. We can then update the steps to be mucchhh simpler.

**Comment by pytlesk4** [comment]
@marbemac what should we do about this article?

**Pull request closed without merging**